containers/wordpress/wp_opt/https/Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG baseimage=wp_http_opt_with_https
+FROM $baseimage
+
+LABEL authors="nitin.tekchandani@intel.com,lin.a.yang@intel.com,uttam.c.pawar@intel.com"
+
+USER root
+
+RUN apt-get update && \
+    apt-get install -y libtool
+
+# Build/Install IPP cyrypto library for the multi-buffer support
```

Comment:
We may want to use a specific branch instead of master
IPP_CRYPTO_VERSION="ippcp_2020u3"

Reply:
done